### ISSUE TITLE
Neat Table Grid

### ISSUE THREAD

**Issue by gibson135** [thoughtbot/neat#377]
Hi. I can't quite figure out how to set this up. I would like to take all of the padding and margin out of everything, at least for some or make a set of both, and it looks like the "Table Grid" setup on the "Bourbon Neat Examples" page (http://neat.bourbon.io/examples/) is what I'm looking for. 

The question really is how this is setup. I'm very new to this, but do understand it in general. 

Are `@include fill-parent();` and `@include row(table);` put in a parent div of the sub-divs (in this case 4 and 8 columns wide) and then in those (the dark blue sub-divs) the @mixins included in them are put in a class added to them? (if that makes sense) 

Basically not sure where the the @mixin's listed in the "Table Grid" example are put or how they are used exactly. 
![table_grid](https://cloud.githubusercontent.com/assets/13741188/9425861/17c70922-48df-11e5-86dc-1f054b126721.jpg)

**Comment by wardpenney** [comment]
Maybe try to just use `@include span-columns(4, $display: block)`

**Comment by tysongach** [comment]
@gibson135 Here’s the source code for that example page, which may help you:

HTML: https://github.com/thoughtbot/neat/blob/gh-pages/examples/index.html#L92 (that example starts on Line 92)
Sass: https://github.com/thoughtbot/neat/blob/gh-pages/_sass/desktop-example.scss#L123 (that example starts on Line 123)

To answer your question, yes, those two side-by-side `div`’s are wrapped with a parent that has:

```scss
@include fill-parent();
@include row(table);
```

Here’s a quick CodePen, which may be easier to grasp since it’s isolated from all of the other examples: http://codepen.io/tysongach/pen/waVPKX?editors=110
